client/state/directly/reducer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DIRECTLY_INITIALIZE
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const isInitialized = createReducer( false, {
+	[ DIRECTLY_INITIALIZE ]: () => true,
+} );
```

Comment:
> I'm very comfortable removing it. In that respect, there's also not really a use case for tracking config at all — should I remove that as well

please! the Redux store should hold the _minimum_ amount of data needed to render the view. anything that can be calculated or derived from more primitive/base data should probably be done so on a function running a render time. that will keep our state smaller and faster due to having fewer reducers and actual state.